### PR TITLE
Synchronize with BCD v5.6.10

### PR DESCRIPTION
**Pull request by OnkarRuikar**
[BCD release v5.6.9](https://github.com/mdn/browser-compat-data/releases/tag/v5.6.9)

### CODE REVIEW

File: files/en-us/web/api/commandevent/command/index.md [hunk 1]
```diff
@@ -3,12 +3,10 @@ title: "CommandEvent: command property"
 short-title: command
 slug: Web/API/CommandEvent/command
 page-type: web-api-instance-property
-status:
-  - experimental
 browser-compat: api.CommandEvent.command
 ---
 
-{{APIRef("Invoker Commands API")}}{{SeeCompatTable}}
+{{APIRef("Invoker Commands API")}}
```
Comment (Josh-Cena) [inline files/en-us/web/api/commandevent/command/index.md:9]: Why is this?

Comment (Josh-Cena) [reply files/en-us/web/api/commandevent/command/index.md:9]: Oh this looks like a BCD bug. `CommandEvent` is marked as experimental but not its properties.

Comment (OnkarRuikar) [reply files/en-us/web/api/commandevent/command/index.md:9]: Sent a PR to fix it https://github.com/mdn/browser-compat-data/pull/24835

File: files/en-us/web/api/invoker_commands_api/index.md [hunk 2]
```diff
@@ -24,12 +24,12 @@ Historically creating these kinds of controls has required JavaScript event list
 
 - [`commandfor`](/en-US/docs/Web/HTML/Element/button#commandfor)
   - : Turns a {{htmlelement("button")}} element into a button, controlling the given interactive element; takes the ID of the element to control as its value.
-- [`command`](/en-US/docs/Web/HTML/Element/button#command)
+- [`command`](/en-US/docs/Web/HTML/Element/button#command) {{experimental_inline}}
```
Comment (Josh-Cena) [inline files/en-us/web/api/invoker_commands_api/index.md:27]: Why is this experimental but not the other?

File: files/en-us/web/api/invoker_commands_api/index.md [hunk 3]
```diff
@@ -24,12 +24,12 @@ Historically creating these kinds of controls has required JavaScript event list
 
 - [`commandfor`](/en-US/docs/Web/HTML/Element/button#commandfor)
```
Comment (OnkarRuikar) [inline files/en-us/web/api/invoker_commands_api/index.md:25]:
```suggestion
- [`commandfor`](/en-US/docs/Web/HTML/Element/button#commandfor) {{experimental_inline}}
```

For now, I am manually fixing it. I'll investigate the script later.

File: files/en-us/web/api/invoker_commands_api/index.md [hunk 4]
```diff
@@ -22,14 +22,14 @@ Historically creating these kinds of controls has required JavaScript event list
...
+- [`command`](/en-US/docs/Web/HTML/Element/button#command) {{experimental_inline}}
   - : Specifies the action to be performed on an element being controlled by a control `<button>`, specified via the `commandfor` attribute.
 
 ## Interfaces
 
-- {{domxref("CommandEvent")}}
+- {{domxref("CommandEvent")}} {{experimental_inline}}
   - : Represents an event notifying the user that a command has been issued. It is the event object for the {{domxref("HTMLElement/command_event", "command")}} event. The event fires on element referenced by {{domxref("HTMLButtonElement.commandForElement", "commandForElement")}}.
 
 ## Extensions to other interfaces
```
[the remark this answers is not in the document]
Comment (OnkarRuikar) [reply files/en-us/web/api/invoker_commands_api/index.md:35]: We've discussed this before. Past discussions about how to derive the status of landing pages with multiple BCD queries never concluded, and we never decided how to handle pages with multiple BCD queries. So, the bot uses only the first BCD query.

**Review by Josh-Cena — APPROVED** [review]

**Pull request merged**